Sources/PayPalNativeCheckout/PayPalUIInterfaces/CheckoutProtocol.swift

```diff
@@ -25,11 +25,9 @@ protocol CheckoutProtocol {
 
 extension Checkout: CheckoutProtocol {
 
-    static func set(config: CoreConfig, returnURL: String) {
+    static func set(config: CoreConfig) {
         let nxoConfig = CheckoutConfig(
-            clientID: config.clientID,
-            returnUrl: returnURL,
-            environment: config.environment.toNativeCheckoutSDKEnvironment()
```

Comment:
Does NXO not need the environment from us anymore?

Reply:
fixed, it does need environment